cmd/apmtelemetrygen/main.go

```diff
@@ -138,6 +138,7 @@ func newRunCmd() *cobra.Command {
 	cmd.Flags().StringVar(&options.EventRate, "event-rate", "0/s", "Must be in the format <number of events>/<time>. <time> is parsed")
 	cmd.Flags().IntVar(&options.Iterations, "iterations", 1, "The number of times to replay the canned data for")
 	cmd.Flags().BoolVar(&options.IgnoreErrors, "ignore-errors", false, "Ignore HTTP errors while sending events")
+	cmd.Flags().BoolVar(&options.EnableRewrites, "do-rewrites", true, "Enable or disable rewriting IDs and timestamps of stored events. When disabled the will replay stored events without any modification. This removes any variance in output.")
```

Comment:
Would it make sense to just wrap the existing flags under the hood and expose two configs? I feel it reduces mental overhead to not have to learn another flag, and it allows more granular usage in case someone only wants to rewrite timestamps.

Reply:
Thus having `-rewrite-ids` and `-rewrite-timestamps`? I would have considered it more complicated (as now there are more possible outputs), but they are straightforward enough that I don't see it as a major issue. I'll amend the PR

Reply:
Amended in [`b765475` (#191)](https://github.com/elastic/apm-perf/pull/191/commits/b7654752ea47c11e19dbfc764e5e6c3c15a4e2f8)